### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by jeremymv2**
Signed-off-by: Jeremy J. Miller <jm@chef.io>

### Description

This is an attempt to make it easier for consumers of the cookbook to understand the necessary config options for their desired scenario.

The config_matrix.html can be viewed [here](http://htmlpreview.github.io/?https://github.com/jeremymv2/audit-docs/blob/master/grid.html)

### Check List

- [x] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [x] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

File: config_matrix.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    chef-server.rb:
+      data_collector['root_url'] = 'https://automate-server.test/data-collector/v0/'
+    </pre>
+  </td>
+</tr>
+<tr>
+  <th>Fetch From Compliance via Chef Server</th>
+  <td> <!-- Fetch from Compliance via Chef Server : Report Directly to Compliance -->
+    <pre>
+    ['audit']['reporter'] = 'chef-compliance'
```
Comment (chris-rock) [inline config_matrix.html]: That should be `chef-server-compliance`

File: config_matrix.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    ['audit']['server'] = 'https://compliance-server.test/api'
+    ['audit']['refresh_token' OR 'token'] = '..'
+    ['audit']['owner'] = 'Owner/Org'
+
+    Compliance Integrated w/ Chef Server
+    </pre>
+  </td>
+  <td> <!-- Fetch from Compliance via Chef Server : Report Directly to Automate -->
+    <pre>
+    ['audit']['reporter'] = 'chef-automate'
```
Comment (chris-rock) [inline config_matrix.html]: `chef-server-automate`

File: config_matrix.html (hunk 1 above)
Comment (jeremymv2) [inline config_matrix.html]:
```
   58   <td> <!-- Fetch from Compliance via Chef Server : Report Directly to Compliance -->
   59     <pre>
   60     ['audit']['reporter'] = 'chef-compliance'
   61     ['audit']['fetcher'] = 'chef-server'
   62     ['audit']['server'] = 'https://compliance-server.test/api'
   63     ['audit']['refresh_token' OR 'token'] = '..'
   64     ['audit']['owner'] = 'Owner/Org'
   65
   66     Compliance Integrated w/ Chef Server
   67     </pre>
   68   </td>
```
Hey @chris-rock I think it's correct as-is.  That section of config is handling: fetching from Compliance via Chef Server (Integration) and report directly to Compliance.  The `fetcher` is therefore chef-server (to fetch via the Integration) and `reporter` is chef-compliance to report directly to Compliance - not via Chef-Server Integration.

**Comment by jeremymv2** [comment]
@chris-rock I can see what it looks like with the html table embedded into the README.md markdown.  My concern initially was that it may be too wide and look strange.   I'll see what happens and paste a screenshot.

**Comment by chris-rock** [comment]
If we have markdown, we can also create a separate docs dir if required. This would allow us to embed this in our inspec page eventually.

File: config_matrix.html (hunk 2 above)
Comment (jeremymv2) [inline config_matrix.html]:
```
69   <td> <!-- Fetch from Compliance via Chef Server : Report Directly to Automate -->
   70     <pre>
   71     ['audit']['reporter'] = 'chef-automate'
   72     ['audit']['fetcher'] = 'chef-server'
   73     ['audit']['server'] = 'https://compliance-server.test/api'
   74     ['audit']['refresh_token' OR 'token'] = '..'
   75     ['audit']['owner'] = 'Owner/Org'
   76
   77     Compliance Integrated w/ Chef Server
   78
   79     client.rb:
   80       data_collector['server_url'] = 'https://automate-server.test/data-collector/v0/'
   81       data_collector['token'] = '..'
   82     </pre>
   83   </td>
```
Again, I think it's correct as-is.  use fetcher=chef-server to fetch via Integration, use report=chef-automate to report Directly to Automate, not via chef-server integration.

**Comment by jeremymv2** [comment]
The new format of the table in the README.md can be seen here: https://github.com/jeremymv2/audit-docs/blob/master/grid3.md

**Comment by jeremymv2** [comment]
@chris-rock  I think this is better now because it is all contained in the README.  I believe these changes make the documentation more streamlined and easier to consume.  Can you review again please?

**Pull request merged**